sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilders/QueryBuilderSamples.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+namespace Azure.DigitalTwins.Core.Samples
+{
+    /// <summary>
+    /// Samples for QueryBuilder.
+    /// </summary>
+    public static class QueryBuilderSamples
+    {
+        /// <summary>
+        /// Main method.
+        /// </summary>
+        public static void Main()
+        {
+            #region Snippet:DigitalTwinsQueryBuilderNonGeneric
+            new DigitalTwinsQueryBuilder();
+            new DigitalTwinsQueryBuilder<BasicDigitalTwin>();
+
+            // SELECT * FROM DigitalTwins
+            new DigitalTwinsQueryBuilder().GetQueryText();
```

Comment:
Do we want to encourage the Build() method call? 
if so, we should reflect that in our samples.